### ISSUE TITLE
[tbendfelt on 2015-10-19 14:41:52] : pDAQ: StringHub - Decrease DOM watchdog timeout sensitivity

### ISSUE THREAD

**Issue by dglo** [dglo/StringHub#267]
Minocqua test runs 126958-126960 have a lot of dropped DOMS.  Analysis shows that the drops are caused by java GC pauses.  The sudden appearance of this phenomenon indicates that Minocqua may be a shade slower initializing the full PDAQ data path which puts more memory pressure on StringHub.

The watchdog timeout should be increased to match the worst-case GC pause.  There is no sense in having a 2GB memory heap if it can not be fully utilized.

**Comment by dglo** [comment]
[tbendfelt on 2015-10-20 12:17:53]
The timeout was increased to 10 seconds and made configurable through a property.  I also bumped up the processor queue depth since my load testing indicated that it can load past 1000 during startup.

------------------------------------------------------------------------
r15825 | bendfelt | 2015-10-20 12:08:10 -0500 (Tue, 20 Oct 2015) | 1 line
Changed paths:
   M /projects/StringHub/trunk/src/main/java/icecube/daq/domapp/NewDataCollector.java

Issue 7589: Increase the watchdog timeout to accommodate a full gc event.
------------------------------------------------------------------------
r15826 | bendfelt | 2015-10-20 12:11:13 -0500 (Tue, 20 Oct 2015) | 1 line
Changed paths:
   M /projects/StringHub/trunk/src/main/java/icecube/daq/domapp/LegacyDataCollector.java

Issue 7589: Increase the watchdog timeout to accommodate a full gc event.
------------------------------------------------------------------------
r15827 | bendfelt | 2015-10-20 12:15:28 -0500 (Tue, 20 Oct 2015) | 1 line
Changed paths:
   M /projects/StringHub/trunk/src/main/java/icecube/daq/domapp/dataprocessor/AsynchronousDataProcessor.java

Issue 7589: Increase the processing queue depth.
------------------------------------------------------------------------
